src/snap/grpc_export.cc

```diff
@@ -23,8 +23,8 @@ limitations under the License.
 #include <string>
 
 #include <grpc++/grpc++.h>
-
 #include <json.hpp>
+#include <boost/network/protocol/http/server.hpp>
```

Comment:
Please update readme of snap-plugin-lib-cpp and add boost as one of dependencies, please remember to add information about boost version.